### PR TITLE
Added snmpv3 trap destination resource

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added snmpv3 trap destination resource

### Check List
- [x] New functionality includes sanity testing.
  - [x] All sanity tests pass. (`$ ansible-test sanity`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).

### CODE REVIEW

File: roles/oneview_appliance_device_snmp_v3_trap_destination_facts/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+#### Options
+
+| Parameter     | Required    | Default  | Choices    | Comments |
+| ------------- |-------------| ---------|----------- |--------- |
+| config  |   No  |  | |  Path to a .json configuration file containing the OneView client configuration. The configuration file is optional. If the file path is not provided, the configuration will be loaded from environment variables.  |
+| name  |   No  |  | |  Name of trap destination.  |
+| uri  |   No  |  | |  Uri of trap destination.  |
+| params  |   No  |  | |  List of params to delimit, filter and sort the list of resources.  params allowed: `start`: The first item to return, using 0-based indexing. `count`: The number of resources to return. `filter`: A general filter/query string to narrow the list of items returned. `sort`: The sort order of the returned data set.  |
+
```
Comment (AsisBagga) [inline roles/oneview_appliance_device_snmp_v3_trap_destination_facts/README.md:20]: did we missed to add below here?
## License 
Apache

**Review by AsisBagga — APPROVED** [review]
fixed missing license tag in READMe.md before merging.

File: plugins/modules/oneview_appliance_device_snmp_v3_trap_destination.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+'''
+
+RETURN = '''
+oneview_appliance_device_snmp_v3_trap_destinations:
+    description: Has all the OneView facts about the OneView appliance SNMPv3 Trap Destination.
+    returned: On state 'present'. Can be null.
+    type: dict
+'''
+
+from ansible_collections.hpe.oneview.plugins.module_utils.oneview import OneViewModule, OneViewModuleException, \
```
Comment (SHANDCRUZ) [inline plugins/modules/oneview_appliance_device_snmp_v3_trap_destination.py:110]: \ is present?

**Review by SHANDCRUZ — APPROVED** [review]

File: plugins/modules/oneview_appliance_device_snmp_v3_trap_destination.py (hunk 2 above)
Comment (VenkateshRavula) [reply plugins/modules/oneview_appliance_device_snmp_v3_trap_destination.py:110]: \ is used for line continuation

**Pull request merged**